components/Button.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+
+const Button = styled.button`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 14px 52px;
+  gap: 12px;
+  border: none;
+  border-radius: 55px;
+  width: 100%;
+  color: var(--white);
+  background: var(
+    ${({ isPrimary }) =>
+      isPrimary ? "--primary-gradient" : "--secondary-gradient"}
+  );
+
+  &:hover {
+    background: var(
+      ${({ isPrimary }) => (isPrimary ? "--primary" : "--secondary")}
+    );
+  }
```

Comment:
Ich kann keinen Hover-Effekt (auf vercel) erkennen.

Reply:
@Roland-Hufnagel der Button wird auf der hinzufügen Seite benutzt. Die Component Button wird in diesem Fall nur benutzt um diese weiter zu "vererben" an `ButtonSmall`

Reply:
alles klar